### ISSUE TITLE
rightClick doesn't work but run passed manually

### ISSUE THREAD

**Issue by hang-ren** [DevExpress/testcafe#2056]
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

rightClick doesn't work

### What is the expected behavior?

Works fine

### How would you reproduce the current behavior (if this is a bug)?

Run the script failed but succeed if manually force it.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

https://ezhome-alpha-3d.shejijia.com/?brand=ezhome

Test code

```js

import { Selector } from 'testcafe';

const continueBtn = Selector('button.btn_continuebtn');
const closeWelcomeImg = Selector('img.closewelcome');
const propertyCheck1 = Selector('div.topproperty');
const unitDropdown = Selector('div.statusBar.unselectable.noprint.propertybar > ul > li.unitDropdown.cdropdown > button > span.utext');
const dimensionDropdown = Selector('div.statusBar.unselectable.noprint.propertybar > ul > li.dimensionDropdown.cdropdown > button > span.utext');
const rightButton1 = Selector('#editor > div.rightmenu > div > div.mainland > div:nth-child(1) > div > div > div');

fixture `New Property Bar`
   .page `https://ezhome-alpha-3d.shejijia.com/?brand=ezhome`

test('new_property_bar_canvas', async t => {
    /* Test Check New Property Bar For Canvas */

    await t
        .maximizeWindow()
        .click(continueBtn)
        .click(closeWelcomeImg)
        .wait(2000)
        .expect(propertyCheck1.innerText).contains('属性', 'The item cannot be shown')
        .expect(unitDropdown.exists).ok()
        .expect(dimensionDropdown.innerText).eql('墙内线', 'The item cannot be shown')
        .rightClick('#editor', {offsetX: 960, offsetY: 400})
        .wait(2000)
        .expect(rightButton1.innerText).contains('缩小 (Ctrl+-)', 'The item cannot be shown')

});

```

### Specify your

* operating system: Windows 7
* testcafe version: 0.18.4
* node.js version:v8.9.0

**Comment by churkin** [comment]
I've reproduced the problem.

**Comment by AlexKamaev** [comment]
fixed in https://github.com/DevExpress/testcafe-hammerhead/issues/1658

**Comment by AndreyBelym** [comment]
Hi @hang-ren, I've just published `testcafe@0.20.5` with the fix! Please, update your TestCafe installation, and feel free to reopen the issue if the problem isn't fixed in your environment.

**Comment by hang-ren** [comment]
@AndreyBelym Well done, thanks for the fix, I will verify it.
